extensions/elytron-security-jdbc/deployment/src/test/java/io/quarkus/elytron/security/jdbc/ParametrizedPathsResource.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.elytron.security.jdbc;
+
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+
+/**
+ * @author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
+ *         <br>
```

Comment:
Not sure about the format

Reply:
Same formatting than [here](https://github.com/quarkusio/quarkus/blob/master/extensions/elytron-security/deployment/src/test/java/io/quarkus/security/test/ParametrizedPathsResource.java) Actually it's a copy/paste...